server/routes/QuestionRoutes.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
```

Comment:
You aren't loading these routes in `server/index.js`. You require them like

```
app.use(require('./routes/QuestionRoutes'))
```
before `app.listen()`